Source/Actions/AlertAction.swift

```diff
@@ -39,8 +39,7 @@ public class AlertAction: NSObject {
     - parameter style:           The action's style
     - parameter handler:         An optional closure that is called when the user taps on this action
     */
-    public convenience init(attributedTitle: NSAttributedString?, style: AlertActionStyle,
-        handler: ((AlertAction) -> Void)? = nil)
+    public convenience init(attributedTitle: NSAttributedString?, style: AlertActionStyle, handler: ((AlertAction) -> Void)? = nil)
```

Comment:
I don't think this should be marked as changed now, right? (I'm adhering to a 110 cols limit which is why this was broken up before)